### ISSUE TITLE
enhancement: bring back embedded images in PDF's

### ISSUE THREAD

**Issue by christinestraub** [Unstructured-IO/unstructured-inference#194]
While the fix for the PR that addressed  [#176](https://github.com/Unstructured-IO/unstructured-inference/issues/176) was a big move forward in merging OCR bboxes+text with layout-model bboxes+text, the issue remains that we are still dropping the "Image" element category that was lost in https://github.com/Unstructured-IO/unstructured/pull/1142/files / https://github.com/Unstructured-IO/unstructured-inference/commit/de19ace101632ed5a8b8e0fed62cef0795fd55ac . Actually, in the comment there

>             # Skip extracted images for this purpose, we don't have the text from them and they
>             # don't provide good text bounding boxes.

Users may want the Image bbox info anyway even if there is no text. One thing we are doing correctly now that shouldn't change is full-page embedded Image PDF's. (e.g. the [IRS pdf](https://utic-dev-tech-fixtures.s3.us-east-2.amazonaws.com/pastebin/IRS-form-1987.pdf) in ingest tests).

Extracted bboxes:

**Comment by badGarnet** [comment]
would a reasonable outcome of this request being:
- image elements are preserved in the list of output elements here https://github.com/Unstructured-IO/unstructured-inference/blob/de19ace101632ed5a8b8e0fed62cef0795fd55ac/unstructured_inference/inference/layoutelement.py#L131 without having it go through the merging logic?

**Comment by cragwolfe** [comment]
yes, as long as the image itself is not  the entire page (or worst case,
there is an arg to specify the behaviour).
